config/samples/redhatcop_v1alpha1_pkisecretenginerole.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: redhatcop.redhat.io/v1alpha1
```

Comment:
we should have real examples. I am not sure if I asked everyone else to do it. I will check, but please change this to a real example.

Reply:
why is not real?